src/styles/_fonts.scss

```diff
@@ -74,4 +74,114 @@
     }
 }
 
-// Imports
\ No newline at end of file
+// Imports

```

Comment:
Looks good to me. I think this can be reviewed and decided when we are designing it but we may need to consider having some of the lighter fonts (i.e. 200 / 300) for mobile-design. You'll notice on the recent figma mockups that some of the content is getting quite condensed and changing font-weight _might_ help - likely not however.

Reply:
Agreed. This was more of a starting point based on the fonts / typography that was currently on the style guide. Didn't want to to add unnecessary bulk before we had confirmed exactly what weight. 
I anticipate 300 in  light, and italic.